docs/devices/WXCJKG12LM.md

```diff
@@ -28,13 +28,15 @@ To do this send to `zigbee2mqtt/FRIENDLY_NAME/set` payload `{"operation_mode": "
 
 As the device is sleeping by default, you need to wake it up after sending the bind/unbind command by pressing the reset button once.
 
-When bound to a lamp, the behavior is as follows (for WXCJKG11LM Aqara Opple switch 1 band):
-- left click: turn off
-- right click: turn on
-- left double click: light dim down (by steps of 33%)
-- right double click: light dim up (by steps of 33%)
-- long left click: warm white
-- long right click: cold white
+This device publish only one endpoint (no up/down specific endpoint).
```

Comment:
Could you please elaborate this a bit more? I don't understand what you mean here.

Reply:
I was speaking of [Binding specific endpoint](https://www.zigbee2mqtt.io/information/binding.html#binding-specific-endpoint). This device does not allow to bind 2 target devices whereas it has 2 x 2 switches (2 bands ; "up" and "down").
That's a shame as WXCJKG11LM (1 band) can also bind same commands to 1 target. In that, WXCJKG12LM  2 bands useless in case of binding: the 1 band device does the same.

I could have misunderstood endpoint and/or publishing concept?